libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/ContraIndicatorConfig.java

```diff
@@ -12,4 +12,5 @@ public class ContraIndicatorConfig {
     private Integer detectedScore;
     private Integer checkedScore;
     private String exitCode;
```

Comment:
Can we just delete the old `exitCode`, if we have both in the config now? (and then remove it from the config later)

Reply:
Ok cool

I wasn't sure how it would deal with extra properties (I think it depends on the JSON deserialisation settings), so hoped it might _just work_ - not too hard to do a two-step deploy though.